src/client/components/form-elements/ReviewCollapsiblePanel.jsx

```diff
@@ -68,22 +69,40 @@ class ReviewCollapsiblePanel extends React.Component {
         </div>);
     }
 
+    const allSections = Object.keys(this.props.uiData.sections);
+    const sectionIndexes = allSections.indexOf(currentPath);
+    const prevPath = allSections[sectionIndexes - 1];
+
+
+    if (sectionsVerified) {
+      hiddenSection = (<div></div>);
+    } else {
+      if (this.props.uiData.sections[prevPath].verified || currentPath === '/veteran-information/personal-information') {
+        hiddenSection = (
+          <div id={`collapsible-${this.id}`} aria-hidden="false" className="usa-accordion-content">
```

Comment:
Do we still need the `aria-hidden` attribute now that we're not hiding this anymore?